README.md

```diff
@@ -1,5 +1,32 @@
 ![GW Open Data Workshop](share/ODWlogo.jpeg)
 
-# GW Open Data Workshop
+# Gravitational Wave Open Data Workshop
 
-This repo is under construction
+Welcome to the Gravitational Wave Open Data Workshop.
+Here you will find the support material for the [Gravitational Wave Open Data Workshop](https://learn.gwosc.org) (ODW)
+organized by the [Gravitional Wave Open Science Center](https://gwosc.org) (GWOSC).
+This material consists on several interactive Jupyter notebooks in the [Tutorials folder](./Tutorials/), divided into topics, grouped into folders for each day of the workshop.
+There are also some quiz questions that you are asked to complete at the end of each topic.
+
+Before you dive into the notebooks, we recommend first taking a look at the [setup guide](./setup.md),
+where you will find information to configure the workspace where you will be executing the tutorials.
```

Comment:
I would move this paragraph in the next section (and reduce duplication).

Reply:
I moved the paragraph to the  setup section and edited the former first paragraph to avoid repetition.